saskatoon/harvest/forms.py

```diff
@@ -610,25 +610,24 @@ class Meta:
 
 
 class EquipmentForm(forms.ModelForm):
-    def clean(self):
-        cleaned_data = super(EquipmentForm, self).clean()
-        bool1 = bool(self.cleaned_data['property'])
-        bool2 = bool(self.cleaned_data['owner'])
-        if not (bool1 != bool2):
-            raise forms.ValidationError(
-                _('Fill in one of the two fields: property or owner.')
-            )
-        return cleaned_data
```

Comment:
please don't delete this, this is still used by `EquipmentAdmin` to make sure we don't fill both the `owner` and `property`

Reply:
this `clean()` only makes sense if we keep the `property` field, which we decided to remove for the basic form.

if we want to keep this field in the admin form, should i then create a separate form for `EquipmentAdmin`?

I reckon the ultimate goal is to do an inline form since it will be edited from the org detail page.

Reply:
old `EquipmentForm` returned as `EquipmentAdminForm` in `harvest/admin.py`